gbeaudry321/ser401-NASA-psyche-mission
# Us327 muti page poi gbeaudry

I implemented the carousel plugin for the POI widget and added a close button.

Test:

1. Play Psyche Level
2. Approach gray cube that is to the right of the starting position (when close enough, an outline will appear around the cube, and an "E" will appear above it)
3. Press "E" 
4. Change the page by clicking on the tabs at the bottom
5. press close exit the widget

Expected:

1. The widget has 3 pages (page 1, page 2, page 3)
2. While it is open you should freely be able to move through the pages
3. The close button should close the widget
4. Pressing "E" again should reopen the widget
